db/migrate/20200109190635_create_active_storage_tables.active_storage.rb

```diff
@@ -1,5 +1,5 @@
 # This migration comes from active_storage (originally 20170806125915)
-class CreateActiveStorageTables < ActiveRecord::Migration[5.2]
+class CreateActiveStorageTables < ActiveRecord::Migration[6.0]
```

Comment:
Is it okay to just change the version like that? Or should we squash all the migrations?

Reply:
Yes, as long as we're confident that the 6.0 migration API defaults don't cause any issues for our existing migration code, and I didn't see any issues.

I'm doing this so that we don't one day find ourselves upgrading to Rails 7 and still having migrations from 5.2 that we need to re-evaluate (if they drop support for the 5.2 migration API at that time).  We'll be a little closer.  Since I did the evaluation already, it'll be reduced future tech debt.

I'm fine with us squashing the migrations, but I think that should be part of a separate PR.

Reply:
👍

Reply:
I had the same question! I guess the real test will be when we add the next migration.